### ISSUE TITLE
Couldn't install on windows

### ISSUE THREAD

**Issue by xlivestudios** [sammchardy/python-binance#251]
How i can install this module in windows? It gives errors every try.

**Comment by ShervinDD** [comment]
Please copy and paste the error.

**Comment by XliveStudios** [comment]
```
    copying src\cryptography\hazmat\primitives\twofactor\__init__.py -> build\lib.win-amd64-3.7\cryptography\hazmat\primitives\twofactor
    running egg_info
    writing src\cryptography.egg-info\PKG-INFO
    writing dependency_links to src\cryptography.egg-info\dependency_links.txt
    writing requirements to src\cryptography.egg-info\requires.txt
    writing top-level names to src\cryptography.egg-info\top_level.txt
    reading manifest file 'src\cryptography.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    no previously-included directories found matching 'docs\_build'
    warning: no previously-included files matching '*' found under directory 'vectors'
    writing manifest file 'src\cryptography.egg-info\SOURCES.txt'
    running build_ext
    generating cffi module 'build\\temp.win-amd64-3.7\\Release\\_padding.c'
    creating build\temp.win-amd64-3.7
    creating build\temp.win-amd64-3.7\Release
    generating cffi module 'build\\temp.win-amd64-3.7\\Release\\_constant_time.c'
    generating cffi module 'build\\temp.win-amd64-3.7\\Release\\_openssl.c'
    building '_openssl' extension
    creating build\temp.win-amd64-3.7\Release\build
    creating build\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7
    creating build\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7\Release
    C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.12.25827\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Ic:\users\administrator\appdata\local\programs\python\python37\include -Ic:\users\administrator\appdata\local\programs\python\python37\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.12.25827\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\winrt" /Tcbuild\temp.win-amd64-3.7\Release\_openssl.c /Fobuild\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7\Release\_openssl.obj
    _openssl.c
    build\temp.win-amd64-3.7\Release\_openssl.c(493): fatal error C1083: Cannot open include file: 'openssl/opensslv.h': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Tools\\MSVC\\14.12.25827\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2

    ----------------------------------------
Command "c:\users\administrator\appdata\local\programs\python\python37\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\1\\pip-install-c91bq33f\\cryptography\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\ADMINI~1\AppData\Local\Temp\1\pip-record-8mm3m9uz\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\ADMINI~1\AppData\Local\Temp\1\pip-install-c91bq33f\cryptography\
```

**Comment by AdvancedVision** [comment]
I had the same trouble. The problem is that pip downloads not compatible version of twisted for python37 (he always downloads 17v for python2). It solves by downloading a needed version for your platform from -> https://www.lfd.uci.edu/~gohlke/pythonlibs/#twisted
than type pip install {the straight name of .whl file}
after that you can: pip install python-binance
but its gonna help))))

**Comment by trentandraka** [comment]
The client works for me after downloading Twisted from that link you posted and installing it with pip. I installed Twisted‑18.4.0‑cp37‑cp37m‑win32.whl, since I have python 3.7 installed, so ymmv.

I was also following the steps from [here](https://gist.github.com/derwolfe/434ba30127660781730e) earlier when trying to get Twisted to work (skipping the first two steps since I didn't want to install an older version of Python after I had just installed 3.7), but idk if it affected anything. Worth a try though if it's still breaking.
